scripts/wsrep_sst_xtrabackup-v2.sh

```diff
@@ -947,11 +947,6 @@ read_cnf()
         done
     fi
 
-    # Retry the connection 30 times (at 1-second intervals)
```

Comment:
I'm not sure if we should just remove it. This is used globally, so on the joiner (server) side as well.
As socat man page says:
```
retry=<num>
Number of retries before the connection or listen attempt is aborted. Default is 0, which means just one attempt.
```